test_hyperparameter_clustering.py

```
@@ -0,0 +1,341 @@
+            reviewId=f"hyper-{i:04d}",
+            rating=rating,
+            createdAt=(datetime.now() - timedelta(days=random.randint(1, 30))).isoformat() + "Z",
+            content=content
+        )
+        reviews.append(review)
+    
+    return reviews
+
+def extract_sentences_and_embeddings(reviews):
+    """리뷰에서 문장을 추출하고 임베딩을 생성"""
+    from app.services.review_analyzer import _split_sentences
```

Ruff (0.12.2)

`app.services.review_analyzer._split_sentences` imported but unused

Remove unused import: `app.services.review_analyzer._split_sentences`

(F401)